examples/data-objects/codemirror/src/index.ts

```diff
@@ -15,11 +15,14 @@ import {
 	RuntimeFactoryHelper,
 } from "@fluidframework/runtime-utils";
 import { MountableView } from "@fluidframework/view-adapters";
+
+import React from "react";
+
```

Comment:
nit: is the extra spacing here intentional? Leave it to me to find the one thing prettier doesn't deal with... 😩

Reply:
Prettier doesn't seem to care either way - typically I prefer to use newlines to split into three groups of imports:

```ts
import { thing1 } from "@fluidframework/package1";
import { thing2 } from "@fluidframework/package2";

import { thing3 } from "non-fluid-package1";
import { thing4 } from "non-fluid-package2";

import { thing5 } from "./localReference1";
import { thing6 } from "./localReference2";
```

Not a strong feeling though if you'd prefer the first two groups get merged.

Reply:
Yeah, prettier only condenses multiple line breaks into one, nothing more. I can't wait for the code that looks like this:

```ts
if (condition === true) {

    const foo = bar;

    console.log(`foo: ${bar}`);

}
```

😝